easybuild/tools/package/utilities.py

```diff
@@ -40,7 +40,8 @@
 from vsc.utils.missing import get_subclasses
 from vsc.utils.patterns import Singleton
 
-from easybuild.tools.config import PKG_TOOL_FPM, PKG_TYPE_RPM, build_option, get_package_naming_scheme
+from easybuild.tools.config import PKG_TOOL_FPM, PKG_TYPE_RPM, build_option, \
+    get_package_naming_scheme, log_path
```

Comment:
no need to break the import line here? if it's less than 120 chars, keep it on the same line